docs/security/trust/content_trust.md

```diff
@@ -221,16 +243,16 @@ Once you have a root key on your system, subsequent images repositories
 you create can use that same root key:
 
 ```bash
-$ docker push docker.io/docker/seaside:latest
-The push refers to a repository [docker.io/docker/seaside] (len: 1)
+$ docker push docker.io/<username>/otherimage:latest
+The push refers to a repository [docker.io/<username>/<someimage>] (len: 1)
```

Comment:
Ah, sorry about that. I have changed it, there you go! Also, wanted to ask is something up with tests not passing in Jenkins?